contracts/src/rollup/IRollupCore.sol

```diff
@@ -113,6 +113,13 @@ interface IRollupCore is IAssertionChain {
      */
     function amountStaked(address staker) external view returns (uint256);
 
+    /**
+     * @notice Get the withdrawal address of the given staker
+     * @param staker Staker address to lookup
+     * @return Withdrawal address of the staker
+     */
+    function withdrawalAddress(address staker) external view returns (address);
```

Comment:
this colliding with the `withdrawalAddress` parameter name used in `newStakeOnNewAssertion` and `newStakeWithoutAssertion`

Reply:
1e9b3d4ff474dc8943a8aac2a9fd3f34cf8c4147